src/tests/EdFi.DataManagementService.Tests.E2E/Features/References/UpdateInvalidReferenceValidation.feature.disabled

```diff
@@ -2,31 +2,46 @@ Feature: Invalid Reference Validation
     PUT requests validation for invalid references
 
         Background:
-            Given this Local Education Agency exists
-                  | id                               | localEducationAgencyId | nameOfInstitution | localEducationAgencyCategoryDescriptor | categories                 |
-                  | c709bb13bc0c4967a3ebf51df5aa5aca | 10203040               | Institution Test  | ["Federal operated agency"]            | ["Local Education Agency"] |
-              And this Student Education Organization Association exists
-                  | id                               | educationOrganizationId | studentUniqueId |
-                  | 1b57dcf9955f4bf987cebea2075c0ed6 | 255901                  | 604834          |
-              And this Student CTE Program Association exists
-                  | id                               | beginDate  | educationOrganizationId | educationOrganizationId | programName                    | programTypeDescriptor              | studentUniqueId |
-                  | 9aa43c870d244c939d366fbfd5ecab10 | 2020-06-05 | 255901                  | 255901                  | Career and Technical Education | ["Career and Technical Education"] | 604825          |
-              And this Graduation Plan exists
-                  | id                               | graduationPlanTypeDescriptor   | educationOrganizationId | schoolYear | totalRequiredCredits |
-                  | f0627865bc5847159757dbc15ec5a375 | Career and Technical Education | 255901                  | 2022       | 10.000               |
-              And this Student Section Association exists
-                  | id                               | beginDate  | localCourseCode | schoolId  | schoolYear | sectionIdentifier           | sessionName             | studentUniqueId |
-                  | 2d22aeab7b5c4ccebb0de7edd67844c2 | 2021-08-23 | ALG-1           | 255901001 | 2022       | 25590100102Trad220ALG112011 | 2021-2022 Fall Semester | 604874          |
-
+            Given the system has these School Years
```

Comment:
No spaces when we specify these resources. Also it's not mandatory but we prefer lower camel case, so this should be `schoolYears`. Same with the rest of the tables in this section